src/utils/devtools.ts

```diff
@@ -29,61 +33,68 @@ export const devtools = <T extends object>(proxyObject: T, name?: string) => {
 
   let isTimeTraveling = false
   const devtools = extension.connect({ name })
-  const unsub1 = subscribe(proxyObject, () => {
+  const unsub1 = subscribe(proxyObject, (ops) => {
+    const action = ops
+      .filter(([_, path]) => path[0] !== DEVTOOLS)
+      .flatMap(([op, path]) => `${op}:${path.map(String).join('.')}`)
```

Comment:
From the code, it seems we can use `map` instead. Can you confirm?
```suggestion
      .map(([op, path]) => `${op}:${path.map(String).join('.')}`)
```

Reply:
Using map just messed the output there!